### ISSUE TITLE
The pre-process of QTL-GWAS colocalisation

### ISSUE THREAD

**Issue by ttongyyang** [chr1swallace/coloc#150]
Hi Chris,
I have learned to use the coloc you developed recently. It is a very useful tool. And I got some questions when I use this package.
When we use the coloc.abf() function, it needs full summary data from GWAS and QTL in our analysis region, but in many published studies, the researcher usually filters out non-significant loci (P > 1*10-8) from GWAS data before they conduct the coloc.abf, such as Xiong X, et al., Nat Genet., 2021(PMID: 34211177) and Li L, et al., Nat Genet., 2021(PMID: 34432052).  I would like to ask whether we can filter out these non-significant loci, whether these loci are critical to affect our result in terms of finding causal genes. It would be greatly helpful for me.

Thank you!

Best regards
Yang Tong

**Comment by jiapeiyuan17** [comment]
Hi Chris,

We also noticed the same question. In many high-impact studies on molecular QTLs, researchers use coloc to analyze the colocalization of QTL loci with GWAS traits. Interestingly, many of these studies applied some cutoff to filter the QTL set or the GWAS trait set before the colocalization analysis. On the coloc website, it is recommended NOT to trim the SNP set by significance or MAF and to include ALL SNPs in that region. Since all of these studies applied a significance cutoff for the SNP set, I am curious if this is acceptable for coloc analysis.

Thank you.

Best,
Jiapei

**Comment by chr1swallace** [comment]
I don't think this is right, no. If you mean they filter to include only snps significant for an eqtl and then try and colocalise with the gwas, they are surely making it harder for them to discover "H3" scenarios, because a information about any gwas signal outside the eqtl signal will not be included.  I wouldn't mind filtering on MAF to exclude impractically rare snps, for example.

On the other hand, if they are filtering the regions by minimum p value, and only testing regions with a p < some threshold I think this would be ok.  And this could be an interpretation of the brief Methods text in one of the papers you cite: "GWAS loci (filtered by P < 10−4) that overlap with g-m6As were used for colocalization tests. "

--
https://chr1swallace.github.io<https://chr1swallace.github.io/>

________________________________

**Comment by jiapeiyuan17** [comment]
Thanks for your quick reply. Really helpful!
